app/navigation/OtherOfficialsNavigator.scala

```diff
@@ -97,95 +97,22 @@ class OtherOfficialsNavigator @Inject()(implicit frontendAppConfig: FrontendAppC
-        otherOfficialRoutes.OtherOfficialsSummaryController.onPageLoad()
-      case Some(false) => otherOfficialRoutes.OtherOfficialsPassportController.onPageLoad(CheckMode, index)
-      case _ =>  routes.SessionExpiredController.onPageLoad()
-    }
-
-    case OtherOfficialsNinoPage(index) => userAnswers: UserAnswers => userAnswers.get(OtherOfficialsNinoPage(index)) match {
-      case Some(_) => otherOfficialRoutes.OtherOfficialsSummaryController.onPageLoad()
-      case _ =>  routes.SessionExpiredController.onPageLoad()
-    }
-
-    case OtherOfficialsPassportPage(index) => userAnswers: UserAnswers => userAnswers.get(OtherOfficialsPassportPage(index)) match {
-      case Some(_) => otherOfficialRoutes.OtherOfficialsSummaryController.onPageLoad()
-      case _ =>  routes.SessionExpiredController.onPageLoad()
-    }
-
-    case OtherOfficialAddressLookupPage(index) => userAnswers: UserAnswers => userAnswers.get(OtherOfficialAddressLookupPage(index)) match {
-      case Some(_) => otherOfficialRoutes.IsOtherOfficialsPreviousAddressController.onPageLoad(CheckMode, index)
-      case _ => routes.SessionExpiredController.onPageLoad()
-    }
-
-    case IsOtherOfficialsPreviousAddressPage(index) => userAnswers: UserAnswers => userAnswers.get(IsOtherOfficialsPreviousAddressPage(index)) match {
-      case Some(true) if userAnswers.get(OtherOfficialPreviousAddressLookupPage(index)).isDefined =>
-        otherOfficialRoutes.OtherOfficialsSummaryController.onPageLoad()
-      case Some(true) => addressLookupRoutes.OtherOfficialsPreviousAddressLookupController.initializeJourney(index, CheckMode)
-      case Some(false) => otherOfficialRoutes.OtherOfficialsSummaryController.onPageLoad()
-      case _ =>  routes.SessionExpiredController.onPageLoad()
-    }
-
-    case OtherOfficialPreviousAddressLookupPage(index) => userAnswers: UserAnswers => userAnswers.get(OtherOfficialPreviousAddressLookupPage(index)) match {
-      case Some(_) => otherOfficialRoutes.OtherOfficialsSummaryController.onPageLoad()
-      case _ => routes.SessionExpiredController.onPageLoad()
-    }
+    case AddedOtherOfficialPage(0) | AddedOtherOfficialPage(1) | AddedOtherOfficialPage(2) => _ =>
+      otherOfficialRoutes.OtherOfficialsSummaryController.onPageLoad()
 
-    case AddAnotherOtherOfficialPage => userAnswers: UserAnswers => userAnswers.get(AddAnotherOtherOfficialPage) match {
-      case Some(_) => routes.DeadEndController.onPageLoad()// TODO summary page
+    case OtherOfficialsSummaryPage => userAnswers: UserAnswers => userAnswers.get(IsAddAnotherOtherOfficialPage) match {
+      case Some(_) if userAnswers.get(Section8Page).contains(true) => routes.IndexController.onPageLoad()
+      case Some(true) if userAnswers.get(OtherOfficialsId(0)).nonEmpty && userAnswers.get(OtherOfficialsId(1)).nonEmpty =>
+        otherOfficialRoutes.OtherOfficialsNameController.onPageLoad(NormalMode, 2)
+      case _ if userAnswers.get(OtherOfficialsId(0)).nonEmpty =>
```

Comment:
adding a note for the future - as discussed in the call, this creates a situation where a user gets stuck in an infinite loop if official #1 is not filled out correctly (as they keep getting sent to the 2nd official's name page). this will be fixed in another ticket